css/forms.css

```diff
@@ -0,0 +1,234 @@
+.range-wrap {
+  display: flex;
+  align-items: center;
+  position: relative;
+  width: 100%;
+  height: 5.25em;
+  font: 1em/1 arial, sans-serif;
+}
+
+[type=range] {
+  flex: 1;
+  margin: 0;
+  padding: 0;
+  min-height: 1.5em;
+  background: transparent;
+  font: inherit;
+}
+[type=range], [type=range]::-webkit-slider-thumb {
+  -webkit-appearance: none;
+}
+[type=range]::-webkit-slider-runnable-track {
+  box-sizing: border-box;
+  border: none;
+  width: 12.5em;
+  height: 0.25em;
+  background: #ccc;
+}
+[type=range]::-moz-range-track {
+  box-sizing: border-box;
+  border: none;
+  width: 12.5em;
+  height: 0.25em;
+  background: #ccc;
+}
+[type=range]::-ms-track {
+  box-sizing: border-box;
+  border: none;
+  width: 12.5em;
+  height: 0.25em;
+  background: #ccc;
+}
+[type=range]::-webkit-slider-thumb {
+  margin-top: -0.625em;
+  box-sizing: border-box;
+  border: none;
+  width: 1.5em;
+  height: 1.5em;
+  border-radius: 50%;
+  background: var(--indigo);
+}
+[type=range]::-moz-range-thumb {
+  box-sizing: border-box;
+  border: none;
+  width: 1.5em;
+  height: 1.5em;
+  border-radius: 50%;
+  background: #f90;
+}
+[type=range]::-ms-thumb {
+  margin-top: 0;
+  box-sizing: border-box;
+  border: none;
+  width: 1.5em;
+  height: 1.5em;
+  border-radius: 50%;
+  background: #f90;
+}
+[type=range]::-ms-tooltip {
+  display: none;
+}
+[type=range] ~ output {
+  display: none;
+}
+
+
+/* Checkbox   */
+
+.checkbox label {
+  display: inline-block;
+  position: relative;
+  font-weight: 400;
+  padding-left: 2rem;
+  margin-bottom: 0
+}
+
+.checkbox label::before {
+  content: "";
+  display: inline-block;
+  position: absolute;
+  width: 2rem;
+  height: 2rem;
+  top: -3px;
+  left: -4px;
+  margin-top: 0;
+  margin-left: -1rem;
+  border: none;
+  border-radius: .375rem;
+  background-color: rgba(255,255,255,0.6);
+  box-shadow: 0px 2px 8px rgba(0,0,0,0.05);
+  -webkit-transition: border 0.15s ease-in-out, color 0.15s ease-in-out;
+  -o-transition: border 0.15s ease-in-out, color 0.15s ease-in-out;
+  transition: border 0.15s ease-in-out, color 0.15s ease-in-out
+}
+
+.checkbox label::after {
+  position: absolute;
+  width: 1.5rem;
+  height: 1.5rem;
+  left: 0;
+  top: 0;
+  margin-top: 4px;
+  margin-left: -0.7rem;
+  font-size: 9px;
+  color: var(--white)
+}
+
+.checkbox input[type="checkbox"] {
+  opacity: 0
+}
+
+.checkbox input[type="checkbox"]:checked+label::after {
+  font-family: "Font Awesome 5 Free";
+  font-weight: 900;
+  content: "\f00c"
```

Comment:
May I know what this does ?

Reply:
Do you mean `\f00c`? it is to provide Unicode as the "content" to the pseudo element. Here it is to give the `tick` mark when I check the checkbox.

Reply:
Yes. Did not know about this. That's cool. Also I just figured out that I'm not able to check the boxes under categories other than 'Architecture' option. Not sure if there's an issue on my side, just wanted to bring it to your notice.

Reply:
you can directly give the symbol itself, if you are not using fontawesome. like 
`content: "✓"`  or even an emoji `content: "✅"`

Reply:
Regarding the checkbox not working - it was because I gave the same `id` to all checkboxes and it was intentional. So, nothing wrong on your end.